### PR TITLE
[full-ci][ui tests] Switch yarn to pnpm for oC Web tests

### PR DESCRIPTION
**Pull request by dschmidt**
## Description
This PR makes drone use `pnpm` for running e2e and acceptance tests for oC Web, as we changed to `pnpm` there recently and we need to use the same lock file here as in the main repo.

As the `.pnpm-store` lives in the main folder of oC Web, we do not need separate caches for acceptance and e2e tests anymore. This simplifies the drone pipelines a bit.

We could also switch idp and settings services over to using `pnpm` but as that is more disruptive for go developers, I'm not doing that in this PR but will send a followup for that.

## Related Issue

- Related https://github.com/owncloud/web/pull/7835

## Motivation and Context
oC Web switched to pnpm and we wanna share the same lock files.

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Todo:
- [ ] Fix all tests
- [ ] Validate .pnpm-store is actually used 
- [ ] Decide whether we want to switch only oC Web tests to pnpm (required for https://github.com/owncloud/web/pull/7835) or also switch services/idp or services/settings

### CODE REVIEW

**Comment by fschade** [comment]
Good decision, lining it 🤗

**Comment by dschmidt** [comment]
I would like to evaluate what's needed to completely switch.
Afterwards we can discuss whether we want to fully switch for settings and idp apps as well or just for e2e/acceptance tests from web.

**Comment by saw-jan** [comment]
I will look at the CI failure.

File: changelog/1.0.0_2020-12-17/fix-konnectd-build.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ Bugfix: Fix konnectd build
 
 Tags: konnectd
 
-We fixed the default config for konnectd and updated the Makefile to include the `yarn install`and `yarn build` steps if the static assets are missing.
+We fixed the default config for konnectd and updated the Makefile to include the `pnpm install`and `pnpm build` steps if the static assets are missing.
```
[the remark this answers is not in the document]
Comment (dschmidt) [reply changelog/1.0.0_2020-12-17/fix-konnectd-build.md:5]: Right 😅

**Comment by saw-jan** [comment]
Tests are now passing for `settingsUI`. Fixed has been applied by @dschmidt.
`e2e` test is still failing and needs a new web release. Currently, the test is failing due to the latest e2e test code running against old web build.

**Comment by dschmidt** [comment]
Ready to merge from my end.

**Review by kulmann — APPROVED** [review]
Grrrrreat 🐯

**Pull request merged**